From 6e1a9a5ca4a4c45983ecbaec229b0dc3f33a4715 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 7 Feb 2018 19:22:12 +0800
Subject: [PATCH] video/rockchip/lcdc/rk3368_lcdc: fix compile warning

drivers/video/rockchip/lcdc/rk3368_lcdc.c:2198:2: warning: this if
clause does not guard... [-Wmisleading-indentation]

Change-Id: I0cbd3ba10b891b67782b1a105eaac3d473e52527
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/video/rockchip/lcdc/rk3368_lcdc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/rockchip/lcdc/rk3368_lcdc.c b/drivers/video/rockchip/lcdc/rk3368_lcdc.c
index f9903f7fcf69..7ddd64da896f 100644
--- a/drivers/video/rockchip/lcdc/rk3368_lcdc.c
+++ b/drivers/video/rockchip/lcdc/rk3368_lcdc.c
@@ -2200,8 +2200,8 @@ static int rk3368_load_screen(struct rk_lcdc_driver *dev_drv, bool initscreen)
 	    dev_drv->trsm_ops &&
 	    dev_drv->trsm_ops->enable)
 		dev_drv->trsm_ops->enable();
-		if (screen->init)
-			screen->init();
+	if (screen->init)
+		screen->init();
 	/*if (!lcdc_dev->standby)
 		lcdc_msk_reg(lcdc_dev, SYS_CTRL,
 			m_STANDBY_EN, v_STANDBY_EN(0));*/
-- 
2.35.3

